conduct.md

```diff
@@ -14,11 +14,27 @@ testing fork
 * Communication
 * Participation
 * Conduct
-* Decisions
```

Comment:
Travis build error:

conduct.md:16: MD032 Lists should be surrounded by blank lines

I would suggest moving the section from lines 17 - 31 so that it starts where line 36 is

Reply:
Fixed it